lambdas/tsconfig.eslint.json

```diff
@@ -3,5 +3,5 @@
         "noEmit": true
     },
     // Additional paths to lint not included by other tsconfig.json files
```

Comment:
Comments are not valid in JSON files. If you want comments, [change the configuration file format](https://eslint.org/docs/latest/use/configure/configuration-files)

Reply:
This is out of scope for this PR - I changed the filter but the file already exists in this form so we can change the format (and in all the other repos) separately

Reply:
Ah, sorry - I misread this and thought you'd added that line in.

Separate PR!